sqlparser/parse_test.go

```diff
@@ -1345,6 +1345,21 @@ var (
 	}, {
 		input:  "select NULL::text from dual",
 		output: "select null::text from dual",
+	}, { // PostgreSQL & MySQL limit&offset format
+		input:  "select * from dual limit 10 offset 10",
+		output: "select * from dual limit 10 offset 10",
+	}, { // PostgreSQL & MySQL limit&offset format
+		input:  "select * from dual limit 10",
+		output: "select * from dual limit 10",
+	}, { // MySQL format
+		input:  "select * from dual limit 10, 10",
+		output: "select * from dual limit 10, 10",
+	}, { // PostgreSQL format
+		input:  "select * from dual limit all",
+		output: "select * from dual limit all",
```

Comment:
As I understand, currently, all of these syntax constructions are supported by both MySQLDialect and PostgreSQLDialect. Does it make sense to throw some error in case of a specific DB format?

Reply:
agree, added errors and tests for these cases. additionally I found how we can configure verbosity of loggers in parser and tokenizer. I just decreased verbosity of tokenizer for acra in non-debug mode to avoid logging sensitive parts of SQL query which we cannot control. But now I keep in mind this options that may help in debugging and extending parser.